gateway/src/apicast/policy.lua

```diff
@@ -31,7 +40,7 @@ function _M.new(name, version)
         _NAME = name,
         _VERSION = version or '0.0',
     }
-    local mt = { __index = policy }
+    local mt = { __index = policy, __tostring = __tostring, policy = policy }
```

Comment:
@davidor solved it by having a property in the metatable. I think that is best of both worlds.